heterogeneity/Documentation_JV_Summer2022

```diff
@@ -0,0 +1,204 @@
+The goal is to replicate his work and extend it to the entire country. Unfortunately, given the lack of documentation this was not possible during the summer. 
+We were only able to partially replicate the results for California. 
+Most of the work consisted on running the python scripts in the correct order and troubleshoot the possible mistakes and missing information we found on the pythion scripts. 
+Most of the work was made in Python using the Spyder app. 
+
+The topics I worked are the following:
+I)	Creating a working environment in Python 
+II)	Order of the python scripts 
+III)	Input and output of every python script  
+IV)	Running time consuming scripts in the SU cluster.
+V)  Where my work ended.
+I will write a description for each with the proper procedure.
+
+I)	Creating a working environment in Python to be used by all the users 
+
+For a complete description of the necessary packages and the how to set up your local environment see the file "Python Order"
+
+I will include an explanaition on how to set up a general environment to be access by every user. 
+
+2)	Order of the python scripts
+
+The order of the python scripts to be run are detailed in the file "Python Order" 
+
+First, all the python scripts are located in:
+G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\ramosRivera\T-Burk\Python Scripts
+
+To help understand what we are doing is good to start with the intuition of the procedure. 
+First, we are digitalizing the addresses of the labs. 
+Second, we are Geocoding those address to be able to locate them in a map using ArcMap. 
+Third, we are creating the clusters by running simulations. 
+Fourth, we are calculating the agglomerations effects. 
+
+In order to carry out the above four tasks the python scripts should be run in the following order:
+
+###Chain to run:
+Pdf2Jpg.py --> OCR.py --> pngwork.py --> Address_ID.py --> state_code_rep.py --> GeoCoder.py -->  field_org.py --> Shapify.py--> K-function Local_3Stage.py--> stat_calc.py
+                      
+
+
+3)	Input and output of every python script
+
+This section detail the inputs and outputs for every python script to make it easy to understand. 
+The diagram is constructed using the following order: 
+
+input that the script received --> python script--> output that the script produce. 
+
+i) Pdf2Jpg.py
+"G:...ramosRivera\T-Burk\PngData\Scans\1998DART32.pdf" --> Pdf2Jpg.py --> "G:...ramosRivera\T-Burk\PngData\Scans\Page_.jpg"
+
+"G:...\ramosRivera\T-Burk\PngData\ScanData\Page_.jpg" (from Pdf2Jpg.py) --> OCR.py --> "G:...\PngData\OCR_Output_1998\OCR_Results.txt"
+
+ii) pngwork.py 
+This script imports several datasets, some of which are not used in the program. I am only documenting the ones that are being used. 
+
+"G:...\T-Burk\PngData\cattell-all.dta"       --> pngwork.py --> 
+"G:...\T-Burk\PngData\Cattell_corr_list.dta" --> pngwork.py --> "G:...\T-Burk\PngData\cattLabs97.csv"
+
+I do not know where the dataset "Cattell_corr_list.dta" is created. 
```

Comment:
@JorgeValde @Kirs10-Riley has this in her notes--essentially, it was emailed to me by Ivan Png. But please get the more detailed description from Kirsten